themes-default/slim/src/components/subtitle-search.vue

```diff
@@ -0,0 +1,258 @@
+                            languages: response.data.languages
+                        });
+                    }
+                })
+                .catch(error => {
+                    console.log(`Error trying to search for subtitles. Error: ${error}`);
+                })
+                .finally(() => {
+                    // Destroy this component.
+                    this.loadingMessage = '';
+                    this.loading = false;
+                    this.$destroy();
+                });
+        },
+        manualSearch() {
+            const { show, season, episode } = this;
+
+            this.displayQuestion = false;
+            this.loading = true;
+            this.loadingMessage = 'Searching for subtitles... ';
+            const url = `home/manualSearchSubtitles?indexername=${show.indexer}&seriesid=${show.id[show.indexer]}&season=${season}&episode=${episode}`;
+            apiRoute(url) // eslint-disable-line no-undef
+                .then(response => {
+                    if (response.data.result === 'success') {
+                        this.subtitles.push(...response.data.subtitles);
+                    }
+                }).catch(error => {
+                    console.log(`Error trying to search for subtitles. Error: ${error}`);
+                    this.$destroy();
+                }).finally(() => {
+                    this.loading = false;
+                });
+        },
+        pickSubtitle(subtitleId) {
+            // Download and save this subtitle with the episode.
+            const { show, season, episode } = this;
+
+            this.displayQuestion = false;
+            this.loadingMessage = 'downloading subtitle... ';
+            this.loading = true;
+            const url = `home/manualSearchSubtitles?indexername=${show.indexer}&seriesid=${show.id[show.indexer]}&season=${season}&episode=${episode}&picked_id=${subtitleId}`;
+            apiRoute(url) // eslint-disable-line no-undef
+                .then(response => {
+                    if (response.data.result === 'success') {
+                        // Update the show, as we have new information (subtitles)
+                        // Let's emit an event, telling the displayShow component, to update the show using the api/store.
+                        this.$emit('update', {
+                            reason: 'new subtitles found',
+                            codes: response.data.subtitles,
+                            languages: response.data.languages
+                        });
+                    }
+                })
+                .catch(error => {
+                    console.log(`Error trying to search for subtitles. Error: ${error}`);
+                })
+                .finally(() => {
+                    // Destroy this component.
+                    this.loadingMessage = '';
+                    this.loading = false;
+                    this.$destroy();
+                });
+        }
+    }
+};
+</script>
+<style>
```

Comment:
You should probably use `<style scoped>`.
I assume the styling doesn't apply to the `vue-good-table` child component when you do?
Try https://vue-loader.vuejs.org/guide/scoped-css.html#child-component-root-elements

For example, you could change:
```css
table.subtitle-table tr {
```
to
```css
.subtitle-search-wrapper >>> table.subtitle-table tr {
```

Reply:
no, I do need to overwrite the vue-good-table styling. Like text color.
For example these two are important:
```
.v--modal-overlay .v--modal-box {
    overflow: inherit!important;
}
table.subtitle-table tr {
    background-color: rgb(190, 222, 237);
}
```

Reply:
Okay, then please try that `>>>` selector with `<style scoped>`. According to the docs it should let you do exactly that, while still keeping the component styles in a scope.

Reply:
never mind. Your suggestion worked. I'm using that.